### DISCUSSION TITLE
Battery power sign always positive

### DISCUSSION THREAD

**Discussion by metcrl** [hultenvp/solis-sensor#272] [Q&A]
Hello, I still seeing the battery power sign always positive. I don't understand, wasn't it fixed with the 3.3.2?
![Screenshot_20230407_215701_Home Assistant](https://user-images.githubusercontent.com/70216900/230669733-575c1d5b-8970-455b-90bc-8162cccf4b99.jpg)
![Screenshot_20230407_215718_Home Assistant](https://user-images.githubusercontent.com/70216900/230669748-8be68fbe-f9ba-451c-8970-10df2acd9c97.jpg)

**Comment by hultenvp** [comment]
Yes, it should have been fixed. Also the slashes were fixed. I'll check what's going on.

**Reply by hultenvp** [reply]
Bit late response, bit the only way this still can be an issue is if the bat current also has a sign issue.
Until now the story is that bat current always had the correct sign, so I copy over the sign of current to power
